src/Resources/config/sylius/twig_hooks/settings/index.yaml

```
@@ -0,0 +1,24 @@
+sylius_twig_hooks:
+    hooks:
+        'monsieurbiz_admin.setting.index.content.header':
```

```suggestion
        'monsieurbiz_admin.settings.index.content.header':
```

Do we fix this on all twig hooks or do we keep it this way?

---

I feel strange about the `monsieurbiz_admin` part also.

---

Wouldn't `monsieurbiz_settings.admin.index.content.header` be a better naming?

---

I was inspired by an official plugin: invoicing. Example https://github.com/Sylius/InvoicingPlugin/blob/2.0/config/twig_hooks/admin/invoice/show.yaml#L3

I will discuss it with the others to see what name we keep for our plugins

---

I vote for the same naming as Sylius Invoicing

---

To me, it will be `sylius_admin.monsieurbiz_settings.index.content.header` if we want to follow Invoicing plugin pattern

---

Or for a shorter version: `sylius_admin.settings.index.content.header` but it could introduce some confusion if there's no indication of the vendor.